scripts/travis/install.sh

```diff
@@ -12,12 +12,9 @@ echo "Install Python dependencies"
 ./peep.sh install -r requirements/default.txt
 echo
 
-# Installing dependencies for smoke tests
-if [[ $TEST_SUITE == "smoke" ]]; then
-  virtualenv venv_smoketests
-  source venv_smoketests/bin/activate
-  pip install -r smoketests/requirements.txt
-  deactivate
+# Installing dependencies for UI tests
+if [[ $TEST_SUITE == "ui" ]]; then
+  ./peep.sh install -r requirements/test.txt
```

Comment:
There was a discussion or something like that to install the smoketests dependencies in a separate virtualenv. I strongly believe that that would be better